omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -90,8 +91,15 @@
 
 <form method="get" id="patient-search-form" onsubmit="return false">
     <input type="text" id="patient-search" placeholder="${ ui.message("coreapps.findPatient.search.placeholder") }" autocomplete="off" <% if (doInitialSearch) { %>value="${doInitialSearch}"<% } %>/><i id="patient-search-clear-button" class="small icon-remove-sign"></i>
+    <div id="patient-search-filter" <%  if (config.hideFilters) { %>hidden<% } %>>
+	    <input type="number" id="filter-age" placeholder="${ ui.message('coreapps.age') }"  onchange="jQuery('#patient-search').focus();"/>
```

Comment:
@kaweesi I'd like to suggest triggering submission (and launch the search) when the user presses enter on the age field (or changes the gender)